src/docs/pages/introduction/getting-started/index.md

```diff
@@ -0,0 +1,221 @@
+</ld-notice>
+
+## Prerequisites
+
+Liquid Oxygen is easy to integrate into an existing frontend project. We assume you already have a project set up. If not, you can create a new project with [Vite](https://vitejs.dev/).
+
+```sh
+npm create vite@latest your-project-name -- --template vanilla-ts
+```
+
+For more information about this command and Vite, please refer to the [Vite documentation](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).<br />Although this guide assumes your project is using typescript, all examples should also be applicable to javascript projects.
+
+## Install
+
+Add Liquid Oxygen to your project with the package manager of your choice.
+
+```sh
+npm install @emdgroup-liquid/liquid
+```
+
+<ld-notice>
+  Liquid Oxygen works with any package manager (e.g. <code>npm</code>, <code>yarn</code> or <code>pnpm</code>). For simplicity, we use <code>npm</code> in this guide.
+</ld-notice>
+
+### Import and register Web Components
+
+The Web Components need to be registered before they can be used. Liquid Oxygen provides the `defineCustomElements` function to register all Liquid Oxygen components. The script loads components lazily only when used in your application. This reduces your bundle size and only loads required code at runtime.
+
+Add following code to your `main.ts` file or any similar file which is loaded for every page.
+
+```js
+// main.ts
+import { defineCustomElements } from '@emdgroup-liquid/liquid/dist/loader'
+defineCustomElements()
+```
+
+#### Avoid lazy loading
+
+If your project requires immediate loading of components, you can use Liquid Oxygen Web Components imported from `@emdgroup-liquid/liquid/dist/components/*`. These components are loaded and automatically registered when imported. This behavior is particularly useful when you are using a bundler like [Vite](https://vitejs.dev/) or [Webpack](https://webpack.js.org/). 
+
+Import the Web Component individually whenever you use it in your application.
+
+```js
+//main.ts
+import "@emdgroup-liquid/liquid/dist/components/ld-button"
+import "@emdgroup-liquid/liquid/dist/components/ld-icon"
+```
+
+<!-- Learn more about the differences in the [Stencil docs](https://stenciljs.com/docs/output-targets). -->
+
+### Import stylesheets
+
+We recommend importing Liquid Oxygen stylesheets in a central place respectively the entry file of your application.
+
+#### Web Components
+
+As Liquid Oxygen Web Components embed their styles, it is sufficient to import the global styles:
+
+```js
+// main.ts
+import '@emdgroup-liquid/liquid/dist/css/liquid.global.css'
+```
+
+#### CSS Components
+
+For the CSS Components, you also need to import the global styles and the styles of the components you are using:
+
+```js
+// main.ts
+import '@emdgroup-liquid/liquid/dist/css/liquid.global.css'
+import '@emdgroup-liquid/liquid/dist/css/ld-button.css'
+```
+
+#### PurgeCSS
```

Comment:
```suggestion
##### PurgeCSS
```

Reply:
hmmm basically a good point. It's not exactly a 3rd option beside web components and css components but it is also not subelement of web components only. 🤔

To be frank, I'd prefer to keep it on this level.